packages/core/src/components/table/table-component-expandable-rows.stories.tsx

```diff
@@ -124,6 +124,14 @@ export default {
       },
       if: { arg: 'noMinWidth', eq: true },
     },
+    overflow: {
+      name: 'Overflow',
+      description: 'Controls the overflow behavior of the expandable row content',
+      control: {
+        type: 'radio',
+      },
+      options: ['scroll', 'hidden'],
```

Comment:
Could we update this to reflect the updated prop alternative 'auto'?

Reply:
Thanks for pointing this out Nathalie :)